teb_local_planner/src/teb_local_planner_ros.cpp

```diff
@@ -493,8 +493,10 @@ geometry_msgs::msg::TwistStamped TebLocalPlannerROS::computeVelocityCommands(
   planner_->visualize();
   visualization_->publishObstacles(obstacles_);
   visualization_->publishViaPoints(via_points_);
-  visualization_->publishGlobalPlan(global_plan_);
-  
+  if (time_last_published_global_plan_ + cfg_->performance.global_plan_publish_freq < nh_->now().seconds()){
```

Comment:
the parameter is named frequency, but actually it's used like an interval.

Reply:
Ah, yes, fixed it :+1: